tests/test_filters.py

```diff
@@ -135,6 +135,19 @@ def test_ascii_hex_decode_missing_eod():
     assert exc.value.args[0] == "Unexpected EOD in ASCIIHexDecode"
 
 
+@pytest.mark.enable_socket()
+def test_decode_ahx():
+    """
+    Cf #1979
```

Comment:
```suggestion
    See #1979
```

Reply:
I typically try to avoid abbreviations. Although most people likely know what they mean, it raises the barrier to contribute for non-native speakers.